Sources/SwiftDocC/Infrastructure/Workspace/DefaultAvailability.swift

```diff
@@ -36,53 +44,112 @@ public struct DefaultAvailability: Codable, Equatable {
         enum CodingKeys: String, CodingKey {
             case platformName = "name"
             case platformVersion = "version"
+            case unavailable
+        }
+        
+        /// The diferent availability states that can be declared.
+        /// Unavailable or Available with an introduced version.
+        internal enum State: Hashable {
```

Comment:
Does `AvailabilityInformation` works? `VersionInformation` sound like there's going to be a version number when it could also be unavailable.

Reply:
Yes, it works. 

I feel that it's not _as_ descriptive since this is already a nested type inside of `DefaultAvailability.ModuleAvailability` so it's _almost_ as descriptive as just `Information` except it doesn't make the reader wonder what type of information it is. I viewed `VersionInformation` a little bit like an optional value with a named `none` case. Either there is version information or there isn't.